leap/outcome_matrix.py

```diff
@@ -39,6 +39,8 @@ def increment(self, column: str, filter_columns: dict | None = None, amount: flo
             for key, value in filter_columns.items():
                 df_filtered = df_filtered.loc[(df_filtered[key] == value)]
             df_filtered[column] += amount
+            # This causes test_run_simulation_one_year to fail
```

Comment:
Glad you caught the bug! These comments should be removed though, it's not recommended to leave commented code